.github/actions/build-rpm/Dockerfile

```diff
@@ -8,7 +8,7 @@ RUN npm install && \
 
 FROM centos:latest
 # library need to generate rpm package
-RUN yum install -y rpm-build rpmdevtools gcc rpm-sign pinentry && \
+RUN yum install -y rpm-build rpmdevtools gcc rpm-sign pinentry make && \
```

Comment:
[Q] Why do we need this additional `make` ?

Reply:
removed